DESCRIPTION

```diff
@@ -52,11 +52,12 @@ Suggests:
     usethis,
     withr
 Config/Needs/check: anthonynorth/roxyglobals
+Config/Needs/website: tidyverse/tidytemplate, dbplyr, Lahman
```

Comment:
Only for the website? Or also for the examples?

Reply:
Only for the website, but I had forgotten to ensure the example would be rendered on the website: [`c594e15` (#359)](https://github.com/tidyverse/duckplyr/pull/359/commits/c594e1578167357cd04aef11e2a59adb0dadf0a9)